src/ansi-c/parser.y

```diff
@@ -1837,13 +1837,21 @@ enum_name:
         }
         ;
 
+basic_type_name_list:
+    basic_type_name
+  | basic_type_name_list basic_type_name
```

Comment:
:information_source: We probably want something like the following, in order to actually build the list - 
```
  | basic_type_name_list basic_type_name
  {
    $$=merge($1, $2);
  }
  ;
```
I have added this :information_source: comment as a note about future work which will likely be needed eventually. This is not a blocking comment.

Reply:
Yes, actually fully supporting this feature is in the pipeline, just getting this. to parse is higher priority though so it comes first.